
gps:
  save_logs: False
  log_level: 20
                  
gps_publisher:
  save_logs: False
  log_level: 20

base:
  save_logs: True
  log_level: 20

rover:
  save_logs: False
  log_level: 20

pointperfect:
  save_logs: False
  log_level: 20